src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -258,6 +265,15 @@ private static Job GetBaseJob(CommandLineOptions options, IConfig globalConfig)
             if (options.RunOncePerIteration)
                 baseJob = baseJob.RunOncePerIteration();
 
+            if (options.EnvironmentVariables.Any())
+            {
+                baseJob = baseJob.With(options.EnvironmentVariables.Select(text =>
+                {
+                    var separated = text.Split(EnvVarKeyValueSeparator);
```

Comment:
Perhaps `text.Split(EnvVarKeyValueSeparator, 2)` to allow `:` to be used in the value?